helm/ovn-kubernetes/crds/k8s.cni.cncf.io_multi-networkpolicies.yaml

```
@@ -0,0 +1,909 @@
+          description: "MultiNetworkPolicy is a CRD schema to provide NetworkPolicy
+            mechanism for net-attach-def which is specified by the Network Plumbing
+            Working Group. MultiNetworkPolicy is identical to Kubernetes NetworkPolicy,
+            See: https://kubernetes.io/docs/concepts/services-networking/network-policies/ ."
+          properties:
+            spec:
+              description: 'Specification of the desired behavior for this MultiNetworkPolicy.'
+              properties:
+                egress:
+                  description: "List of egress rules to be applied to the selected pods.
+                    Outgoing traffic is allowed if there are no NetworkPolicies selecting
+                    the pod (and cluster policy otherwise allows the traffic), OR if the
+                    traffic matches at least one egress rule across all of the NetworkPolicy
+                    objects whose podSelector matches the pod. If this field is empty
+                    then this NetworkPolicy limits all outgoing traffic (and serves solely
+                    to ensure that the pods it selects are isolated by default). This
+                    field is beta-level in 1.8"
+                  items:
+                    description: "NetworkPolicyEgressRule describes a particular set of
+                      traffic that is allowed out of pods matched by a NetworkPolicySpec's
+                      podSelector. The traffic must match both ports and to. This type
+                      is beta-level in 1.8"
+                    properties:
+                      ports:
+                        description: "List of destination ports for outgoing traffic. Each
+                          item in this list is combined using a logical OR. If this field
+                          is empty or missing, this rule matches all ports (traffic not
+                          restricted by port). If this field is present and contains at
+                          least one item, then this rule allows traffic only if the traffic
+                          matches at least one port in the list."
+                        items:
+                          description: "NetworkPolicyPort describes a port to allow traffic on"
+                          properties:
+                            port:
+                              anyOf:
+                                - type: integer
+                                - type: string
+                              description: "The port on the given protocol. This can either
+                                be a numerical or named port on a pod. If this field is
+                                not provided, this matches all port names and numbers."
+                              x-kubernetes-int-or-string: true
+                            endPort:
+                              type: integer
+                              description: "If set, indicates that the range of ports from
```

```suggestion
                            endPort:
                              type: integer
                              format: int32
                              description: "If set, indicates that the range of ports from
```
@cathy-zhou CRD is not latest, see:  https://github.com/k8snetworkplumbingwg/multi-networkpolicy/blob/master/scheme.yml#L504